### ISSUE TITLE
Feature: allow specifying a prefix for the cache path

### ISSUE THREAD

**Issue by alexrjones** [tehlers/gradle-gcs-build-cache#13]
Thanks for your plugin! It works well.

It would be useful to be able to optionally specify a path where the cache will be stored within the bucket. For instance, if I specify 

```groovy    
remote( GCSBuildCache.class ) {
      bucket = 'my-bucket'
      prefix = 'app-cache'
}
```

then cache entries will be written to and read from `gs://my-bucket/app-cache/`.

**Comment by cpapad** [comment]
Hello @tehlers! Nice work! :)

Good you release a new tag with this feature so that we can import it properly to our project?

Thank you in advance

**Comment by tehlers** [comment]
Hi @cpapad, I've just published release 1.3.0.

**Comment by cpapad** [comment]
> Hi @cpapad, I've just published release 1.3.0.

Thank you!!!
